### PR TITLE
[CMS-1144] Add PHP Compatibility Github Action.

### PR DESCRIPTION
**Pull request by kporras07**

### CODE REVIEW

**Comment by jspellman814** [comment]
Is there a JIRA ticket for this? If so, can you add to the title of the PR?

**Comment by kporras07** [comment]
Done! Thanks

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+name: PHP Compatibility
+on:
+  push:
```
Comment (jspellman814) [inline .github/workflows/ci.yml:3]: Is a branch supposed to be specified here?

Comment (jspellman814) [reply .github/workflows/ci.yml:3]: If we didn't want to limit this action to a branch, the syntax would be `on: push`

Comment (kporras07) [reply .github/workflows/ci.yml:3]: nope, that means runs on push on every branch

Comment (jspellman814) [reply .github/workflows/ci.yml:3]: I think the syntax is wrong as per [the docs](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#using-a-single-event)

Comment (kporras07) [reply .github/workflows/ci.yml]: It's a valid syntax but you're right, let's stick with how it's supposed to be. Updated

**Review by jspellman814 — APPROVED** [review]

**Pull request merged**